# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../spice/linuxconf.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./conf/linux.org	Fri Jun 25 07:24:00 1999
+++ ./conf/linux	Tue May 27 10:18:22 2003
@@ -14,7 +14,7 @@
 
 CC_OPT	        = -O2 -s
 
-SPICE_DIR       = /usr/local/spice
+SPICE_DIR       = /usr/spice
 SPICE_LIB_DIR   = $(SPICE_DIR)/lib
 SPICE_EXEC_DIR  = $(SPICE_DIR)/bin
 
